host/azure-pipelines.yml

```diff
@@ -17,8 +17,45 @@ pr:
     include:
       - host/src/
 stages:
+  - stage: BuildAndPublishPrelaunchApps
```

Comment:
YAML updates to publish /PrelaunchApp.App.csproj for NET7 and NET8 framework and include that in the worker nuget package.

Reply:
I published the preview version of nuget package and that was used to validate the changes using this branch: https://github.com/Azure/azure-functions-host/blob/69bcacc99bd483de2ec42732100a54814d5cb2d6/src/WebJobs.Script/WebJobs.Script.csproj#L50